### ISSUE TITLE
Can't import vispy.scene if matplotlib is installed

### ISSUE THREAD

**Issue by JPFrancoia** [vispy/vispy#1446]
Hi,

I can' import ```vispy.scene``` if matplotlib is installed in my virtualenv:

```
Traceback (most recent call last):
  File "main.py", line 16, in <module>
    from canvas import MyCanvas
  File "/home/djipey/informatique/python/ChemCAD/canvas.py", line 6, in <module>
    import vispy.scene
  File "/home/djipey/.local/share/virtualenvs/cad/lib/python3.6/site-packages/vispy/scene/__init__.py", line 33, in <module>
    from .visuals import *  # noqa
  File "/home/djipey/.local/share/virtualenvs/cad/lib/python3.6/site-packages/vispy/scene/visuals.py", line 18, in <module>
    from .. import visuals
  File "/home/djipey/.local/share/virtualenvs/cad/lib/python3.6/site-packages/vispy/visuals/__init__.py", line 23, in <module>
    from .isocurve import IsocurveVisual  # noqa
  File "/home/djipey/.local/share/virtualenvs/cad/lib/python3.6/site-packages/vispy/visuals/isocurve.py", line 18, in <module>
    from matplotlib import _cntr as cntr
ImportError: cannot import name '_cntr'
```

Uninstalling matplotlib solves the problem, but I need this lib for my project.

EDIT:
commenting line 18 in isocurve.py works as well.

Is there a workaround ?

**Comment by JPFrancoia** [comment]
Actually I just found this module shouldn't be used anymore:

https://stackoverflow.com/questions/49160142/how-should-cntr-should-be-imported-on-matplotlib-lts-2-2-0

**Comment by kmuehlbauer** [comment]
@JPFrancoia There is a hotfix in latest master at commit a87e404753280a9f684ef688553175b072d0eec7 if you need latest matplotlib. Another solution would be to downgrade matplotlib to the version before 2.2.0 i think. 

@davidh-ssec Any plans for  another 0.5.X bug-fix release.

**Comment by davidh-ssec** [comment]
@kmuehlbauer Yes. I was hoping to double check some more stuff with jupyter notebook support before doing it, but keep hitting roadblocks. I would suggest @JPFrancoia as you suggested.

**Comment by JPFrancoia** [comment]
The hotfix is sufficient for my use case (I think), as I don't need Isocurves.

**Comment by davidh-ssec** [comment]
Fixed by #1458
